semgrep_output_v1.atd

```diff
@@ -597,7 +597,7 @@ type sca_match_kind = [
    * access third-party code for further investigation
    * (similar to (LockfileOnlyMatch Transitive))
   *)
-  | TransitiveUndetermined
+  | TransitiveUndetermined of transitive_undetermined
```

Comment:
@aryx I think we will need to override the backwards compatibility check for this, does that seem reasonable? Should be okay since these types aren't consumed anywhere, right?

Reply:
yes it's fine

Reply:
can you override and merge? I don't think I have permission